frontend/src/components/SearchLozenge/SearchLozenge.css

```diff
@@ -0,0 +1,23 @@
+.lozengeBubbles{
+    text-align: center;
+}
+
+.lozenge{
+    float: left;
+    text-align: center;
+    cursor: pointer;
+    border: #888AA0;
+    color: #757575;
+    font-size: 20px;
+    box-shadow: 1.5px 2px 1.5px 2px #E7E7E7;
+    margin: 10px;
+    padding-left: 20px;
+    padding-right: 20px;
+    border-radius: 50px 50px;
+}
+
+button{
```

Comment:
CSS is shared across all the pages, so if you style `button` here, it will style buttons across the entire site. We probably don't want to do that, so you'll want to use a class to handle this.

I'd also probably try to position this over toward the right edge of the lozenge, using border-radius to make sure it doesn't escape the container.

Reply:
Issue fixed!